src/runtime.c

```diff
@@ -404,6 +412,16 @@ main (int argc, char *argv[])
                             }
                         }
                     } else if(inode.base.inode_type == SQUASHFS_REG_TYPE || inode.base.inode_type == SQUASHFS_LREG_TYPE){
+                        if(created_inode[inode.base.inode_number - 1]) {
```

Comment:
I'm not entirely sure why this check is performed. Would you please add a comment or two explaining what you do?